client/src/views/TrackViewer/Viewer.vue

```diff
@@ -100,11 +106,12 @@ export default defineComponent({
       allTypes,
       filteredTrackIds,
       enabledTrackIds,
+      updateTypeName,
     } = useTrackFilters({ trackMap, sortedTrackIds });
 
     // Initialize the view
     Promise.all([
-      loadDataset(datasetId),
+      loadDataset(datasetId).then((meta) => { loadTypeColors(meta && meta.customTypeColors); }),
```

Comment:
I don't think `loadTypeColors` should be part of this try/catch.

If the REST request fails, we want to show the user an error dialog (TODO) but if `loadTypeColors` errors, that should dump to console.  Could this be moved after the catch using `dataset.value.meta` ?